SPECS/scossl/scossl.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+    "Signatures": {
+     "scossl-0.1.tar.gz": "548bfd75a8a03ee9bfb75211d9968eb319bd7b79d03c92d5ebd5c12e5eae4143"
```

Comment:
Can't comment on `libsymcrypt.so`, so adding this here: I think you've accidentally checked-in a library into the repo.:)